src/diem/testing/miniwallet/app/app.py

```diff
@@ -77,9 +78,10 @@ def _balances(self, account_id: str) -> Dict[str, int]:
                 ret[txn.currency] = ret.get(txn.currency, 0) + txn.balance_amount()
         return ret
 
-    def _gen_subaddress(self, account_id: str) -> Subaddress:
+    def _gen_subaddress(self, account_id: str) -> str:
```

Comment:
do we want to change the name to `_gen_subaddress_hex` to remain consistent with the `_hex` suffix everywhere else?

Reply:
changed to return subaddress bytes instead, caller can decide converting to hex as needed.